### ISSUE TITLE
Disturbing warning ``unsupported build info format in ..., building all``

### ISSUE THREAD

**Issue by jfbu** [sphinx-doc/sphinx#4478]
#### Procedure to reproduce the problem

Take any project and issue

```console
sphinx-build -a . _build/ 
sphinx-build . _build/ 
```

The second run issues this Warning

```
WARNING: unsupported build info format in '/foo/bar/_build/.buildinfo', building all
```

#### Expected results

A useful warning or none.

### Environment info

- OS: Mac 10.9.5
- Sphinx version: 1.6.6
- Python version: 3.6.4 (CPython)
- Docutils version: 0.14 
- Jinja2 version: 2.10

**Comment by stephenfin** [comment]
Looks like a dupe of #4477

**Comment by jfbu** [comment]
@stephenfin indeed it does :) quite a coincidence! I should have paid more attention to recent issues. I will mark this as duplicate, as I expect fix of #4477 will also fix this. Thanks for pointing it out!

**Comment by tk0miya** [comment]
Fixed by #4480.
Thank you for reporting
